### PR TITLE
Update UG with new description

### LINKED ISSUE #211: UG: Update application description
The application description in the UG is on AB3 instead of ResearchRoster. The `Features` section does not have a clear order and is just arbitrarily listed.

Let's do the following:
1. Update description to a user-centric description of RR.
2. Segregate the commands into `Basic Commands` and `Advanced Commands`, and create a section table of content.

### PR DESCRIPTION
**Pull request by k-hian**
Fixes #211 

Update the description to a User-centric description of ResearchRoster, and a mini goal-oriented table of content. Segregate the Features section into 'Basic Commands' and 'Advanced Commands', and add a section table of content.

Also update Kay Hian's portfolio with contributions.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -3,7 +3,22 @@ layout: page
 title: User Guide
 ---
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+Welcome to ResearchRoster! **Your Research, Simplified.**
+
+Designed with researchers in mind. ResearchRoster simplifies participant management so you can focus on what matters most: your research. Easily add and delete participants as required, conduct random sampling with a simple command, and export contacts for seamless communication.
```
Comment (PrishaVP) [inline docs/UserGuide.md:8]: I thought "random sampling" is not the correct term for what we are doing in our assign command?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -272,10 +300,12 @@ _Details coming soon ..._
 
 Action | Format, Examples
 --------|------------------
+**Help** | `help`
 **Add** | `add n/NAME e/EMAIL g/GENDER a/AGE [d/DETAIL] [t/STUDY_GROUP_TAG]…​` <br> e.g., `add n/James Ho e/jamesho@example.com g/M a/30 d/to be assigned  t/1A t/3C`
-**Clear** | `clear`
+**List** | `list`
+**Edit** | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g., `edit 2 n/James Lee e/jameslee@example.com`
```
Comment (PrishaVP) [inline docs/UserGuide.md:306]: what about for removing tags?

File: docs/team/kayhian.md [hunk 3]
```diff
@@ -9,38 +9,32 @@ ResearchRoster is a desktop address book application used for teaching Software
...
-  * Highlights: This enhancement affects existing commands and commands to be added in the future. It required an in-depth analysis of design alternatives. The implementation too was challenging as it required changes to existing commands.
-  * Credits: *{mention here if you reused any code/ideas from elsewhere or if a third-party library is heavily used in the feature so that a reader can make a more accurate judgement of how much effort went into the feature}*
+* **New Feature**: Added `assign` command to (randomly) assign persons to study group(s) [\#125]()
+  * What it does: allows the user to randomly assign a list of persons into study groups.
+  * Justification: As our product is targeted at researchers, we anticipate the need to quickly and fairly assign participants to study groups. This feature allows the user to assign an entire list of participants in one command call.
+  * Highlight: a bonus feature of this command is the ability to assign all persons in the displayed list to a single study group by inputting only one staudy group name.
```
Comment (PrishaVP) [inline docs/team/kayhian.md:15]: study is spelt as staudy :(

Comment (k-hian) [reply docs/team/kayhian.md]: Thank you. I have fixed the bugs mentioned.

**Comment by wkxcass** [comment]
LGTM!

**Pull request merged**